atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -139,4 +137,12 @@ public void compactInternally() {
             }
         }
     }
+
+    private String prefixedTableName() {
```

Comment:
Agreed, I spend a little time thinking about it and couldn't find a nice replacement.  Let's leave it as is for the moment.